modules/internal/ChapelError.chpl

```diff
@@ -427,12 +444,18 @@ module ChapelError {
 
     const myFileC:c_string = __primitive("chpl_lookupFilename",
                                          __primitive("_get_user_file"));
-    const myFileS = createStringWithNewBuffer(myFileC);
+    var myFileS: string;
+    try! {
+      myFileS = createStringWithNewBuffer(myFileC);
```

Comment:
This one is interesting. It's presumably possible to have a Chapel program in a filename that is not UTF-8. In that event, this function should probably just escape the non-UTF-8 characters rather than halting.

Reply:
Right. Escaping non UTF8 sequences is not a capability we have right now but we are planning that for other efforts. I think it'd be reasonable to use that here once we have it.